app/controllers/admin/configs_controller.rb

```diff
@@ -96,8 +96,11 @@ def clean_up_params
     end
 
     def bust_relevant_caches
-      # Needs to change when suggested_tags is edited.
-      CacheBuster.bust("/tags/onboarding")
+      CacheBuster.bust("/tags/onboarding") # Needs to change when suggested_tags is edited.
+      CacheBuster.bust("/shell_top") # Cached at edge, sent to service worker.
+      CacheBuster.bust("/shell_bottom") # Cached at edge, sent to service worker.
+      CacheBuster.bust("/onboarding") # Page is cached at edge.
+      Rails.cache.delete_matched(ApplicationConfig["RELEASE_FOOTPRINT"]) # Delete all caches tied to this key.
```

Comment:
Can  `RELEASE_FOOTPRINT` contain any glob patterns? If yes, you need to escape them with `\` to match verbatim. The glob patterns are `?`, `*`, character classes (`[ab]`) and negated character classes (`[^ab]`). Single `.` as in version numbers are ok:

```
❯ redis-cli
127.0.0.1:6379> MSET foo 1 fao 2 f.o 3
OK
127.0.0.1:6379> KEYS f.o
1) "f.o"
```

Reply:
It's set by the environment 

`export RELEASE_FOOTPRINT=$(date -u +'%Y-%m-%dT%H:%M:%SZ')` in `entrypoint.sh`... So no we don't have any glob patterns... Someone could set this otherwise in their environment but that seems like an unwise choice on their part.

I think we should be good on all this.

Reply:
Thanks @benhalpern! I sort of remembered what `RELEASE_FOOTPRINT` looks like, the comment was also for refreshing my own memory of how matching on Redis keys works.